app/styles/partials/footer.scss

```diff
@@ -33,3 +33,68 @@ footer {
     right: 0 !important;
   }
 }
+
```

Comment:
And please don't put these styles in footer.scss. They have no relationship to that.